### ISSUE TITLE
Feature Request: Add `Fork` button to each repository

### ISSUE THREAD

**Issue by ScottRudiger** [refined-bitbucket/refined-bitbucket#299]
I find it inconvenient (and easy to forget) to find the `Fork this repository` button hidden behind the `+` sidebar menu:

<img width="361" alt="Screen Shot 2019-07-14 at 7 57 34 PM" src="https://user-images.githubusercontent.com/26824724/61193668-c8aff200-a671-11e9-9824-e6d0a85b2df5.png">

Could the extension add a `Fork` button on the main page of each repository? Perhaps next to the `Clone` button:

<img width="197" alt="Screen Shot 2019-07-14 at 7 59 10 PM" src="https://user-images.githubusercontent.com/26824724/61193712-044abc00-a672-11e9-81df-03e5103c2faf.png">

**Comment by anikethsaha** [comment]
would like to take this. Is there any platform to have a chat. need to clarify some doubts

**Comment by anikethsaha** [comment]
Is it taken or not ?

**Comment by andremw** [comment]
No, it's not taken. I assigned the issue to help you get in touch with @reyronald and start working on the issue.

**Comment by anikethsaha** [comment]
Ohh okok

**Comment by anikethsaha** [comment]
I looked into the bitbucket code and had a chat with @reyronald , as bitbucket is using CSS-in-js (styled-components), it would be very un-maintainable to place the fork button on top. As I would be referencing an element using clase name cause I don't see any other unique attributes to reference, I would break whenever bitbucket is publishing new version of their frontend. I thought of using `display: fixed`  then placing the button but it will break too if device size changes. 

**Anyone has any workaround or better idea.**

I have made API for checking for source pages and other small things but placing the button is quite difficult in this case
